client/src/pages/ProfileList/profileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box } from '@mui/system';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import useStyle from './useStyles';
+import Avatar from '@mui/material/Avatar';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, Rating, Divider } from '@mui/material';
+function ProfileCard(): JSX.Element {
+  const classes = useStyle();
+  return (
+    <Card sx={{ maxWidth: 300 }}>
+      <CardActionArea>
+        <CardMedia>
+          <Avatar
+            src="https://th.bing.com/th/id/R.ea1b9ab324ce1e392dab4771e9524042?rik=iHtWE4I7L9%2bcaQ&pid=ImgRaw&r=0"
+            alt="green iguana"
+            className={classes.profileImage}
+            sx={{ width: 100, height: 100 }}
+          />
+        </CardMedia>
+        <CardContent className={classes.profileDetails}>
+          <Typography gutterBottom variant="h5" component="div" sx={{ fontWeight: '500', fontSize: 18 }}>
+            Jessica Parker
```

Comment:
As opposed to hard coding data it would be feasible to integrate if we would've contained all the necessary information within an object and used the object's properties

Reply:
Created the object to hold all the properties.